note-app/src/components/NoteList/NoteItem/NoteItem.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Styles } from './styles';
+import { Button } from '@material-ui/core';
+
+function NoteItem({ note, onSelect }) {
+  let totalWrapperStyle;
+
+  if (note.isActive) {
+    totalWrapperStyle = { ...Styles.itemWrapper, ...Styles.itemWrapper_active };
+  } else {
+    totalWrapperStyle = { ...Styles.itemWrapper };
+  }
```

Comment:
I think we can reduce amount of code here:
```suggestion
  const wrapperStyle = { ...Styles.itemWrapper,  ...(!isActive&& {...Styles.itemWrapper_active}) };
```